### PR TITLE
Well analysis trail

### LINKED ISSUE #1067: `WellAnalysis` to WLF
https://webviz-subsurface-example.azurewebsites.net/well-analysis

### PR DESCRIPTION
**Pull request by EirikPeirik**
This PR transfers the `WellAnalysis` plugin to WLF.

I have removed the well attributes filtering for now, plan to re-introduce them when the new AttributeSelector becomes available.

---

### Contributor checklist

- [x] :tada: This PR closes #1067 .
- [ ] :scroll: I have broken down my PR into the following tasks:
   - [ ] Task 1
   - [ ] Task 2
- [ ] :robot: I have added tests, or extended existing tests, to cover any new features or bugs fixed in this PR.
- [x] :book: I have considered adding a new entry in `CHANGELOG.md`, and added it if should be communicated there.

### CODE REVIEW

File: webviz_subsurface/plugins/_well_analysis/_views/_well_overview.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                    for attr_l in well_attr_selected:
+                        for attr in attr_l:
+                            if attr in value:
+                                attr_list.append(attr)
+                    well_attributes_selected[category] = attr_list
+
+            # Make set of wells that match the well_attributes
+            # Well attributes that does not exist in one ensemble will be ignored
+            wellattr_filtered_wells: Set[str] = set()
+            for _, ens_data_model in self.data_models.items():
```
Comment (JinCheng2022) [inline webviz_subsurface/plugins/_well_analysis/_views/_well_overview.py:148]: use list(self.data_models.values() instead?

File: webviz_subsurface/plugins/_well_analysis/_plugin.py [hunk 2]
```diff
@@ -76,18 +77,13 @@ def __init__(
...
-            / "inplace_volumes.css"
-        )
+        self.error_message = ""
 
         self._ensembles = ensembles
         self._theme = webviz_settings.theme
 
         if ensembles is None:
+            self.error_message = 'Incorrect argument, must provide "ensembles"'
             raise ValueError('Incorrect argument, must provide "ensembles"')
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_well_analysis/_plugin.py:87]: This code pattern was new to me. In some way the `self.error_message` gives value even though we on the next line rasies an exception?

Comment (lindjoha) [reply webviz_subsurface/plugins/_well_analysis/_plugin.py:87]: No, this doesn't make sense. This error layout thing is something the summer student has taken from the WLF boilerplate repo. But everywhere else in `webviz-subsurface` the error messages is seen in the terminal so I think I'll just remove that from here.

File: webviz_subsurface/plugins/_well_analysis/_plugin.py [hunk 3]
```diff
@@ -116,24 +107,64 @@ def __init__(
...
+                .get_unique_id(),
+                "content": "Choose chart type, ensemble, type of production and other options. "
+                "You also change the layout of the chart.",
+            },
+            {
+                "id": self.view(self.Ids.WELL_OVERVIEW)
+                .settings_group(WellOverviewView.Ids.FILTERS)
+                .get_unique_id(),
+                "content": """You can choose to view the production for all the wells or
+                                select only the ones you are interested in.""",
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_well_analysis/_plugin.py:140]:
```suggestion
                "content": "You can choose to view the production for all the wells or "
                                "select only the ones you are interested in.",
```

File: webviz_subsurface/plugins/_well_analysis/_plugin.py [hunk 4]
```diff
@@ -116,24 +107,64 @@ def __init__(
...
+            },
+            {
+                "id": self.view(self.Ids.WELL_CONTROL)
+                .view_element(WellControlView.Ids.VIEW_ELEMENT)
+                .component_unique_id(WellControlViewElement.Ids.CHART),
+                "content": """Shows the number of realizations on different control modes.
+                             The control modes are listed in the legend. Also shows
+                             Network pressures according to dates.""",
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_well_analysis/_plugin.py:148]:
```suggestion
                "content": "Shows the number of realizations on different control modes. "
                             "The control modes are listed in the legend. Also shows "
                             " Network pressures according to dates.",
```

**Review by anders-kiaer — APPROVED** [review]
Nice! 🚀 Just a couple of super small suggestions from me. LGTM.

**Pull request merged**